meta-refkit-core/conf/distro/include/refkit-config.inc

```diff
@@ -47,6 +47,9 @@ REFKIT_DEFAULT_DISTRO_FEATURES += "refkit-firewall"
 # Reconfigure several upstream recipes for the computer vision profile.
 REFKIT_DEFAULT_DISTRO_FEATURES += "refkit-computervision"
 
+# Reconfigure/patch upstream recipes for the gateway vision profile.
+REFKIT_DEFAULT_DISTRO_FEATURES += "refkit-gateway"
```

Comment:
This might be better to have in a separate commit. It's not mentioned in the commit message either.